openlcs/libs/driver.py

```diff
@@ -116,7 +117,7 @@ def date_handler(obj):
             return obj.isoformat() if hasattr(obj, 'isoformat') else obj
         return requests.post(
             abs_url, headers=self.headers, data=json.dumps(
-                data, default=date_handler))
+                data, default=date_handler), timeout=10)
```

Comment:
I might have missed the point. But is there a reason you explicitly set timeout here? @hanchuntao

Reply:
For the latest version about pylint, it will check the timeout, such as: https://gitlab.corp.redhat.com/product-security/dev/openlcs/-/jobs/1701509